### DISCUSSION TITLE
Swedish characters in the password from LDAP is not connecting with opengrok 1.7.6 version

### DISCUSSION THREAD

**Discussion by vigneshkarthy3** [oracle/opengrok#4111] [General]
**Describe the bug**
We have swedish passwords for our organization which we use it to connect opengrok the above mentioned version. But, it's only working when we use english characters and numbers and numericals.

**To Reproduce**
change your password with swedish characters, but i dont think you can replicate this because the methods can vary!

**Expected behavior**
Doesn't allow you to get past the username password authentication

**Screenshots**
Difficult as we have NDA.

**Comment by vladak** [comment]
I don't see how this is related to OpenGrok. The webapp does not participate in any sort of authentication.

**Reply by vigneshkarthy3** [reply]
LDAP authentication works for every other applications installed on the server, except opengrok. I even tried changing the locale of the server, it didn't work

**Reply by vladak** [reply]
Again, OpenGrok itself does not participate in any sort of authentication. Given the lack of detail, you will have to debug yourself. Good luck.

**Reply by vigneshkarthy3** [reply]
Okay sure.
